### DISCUSSION TITLE
What operating systems does the Windows app support?

### DISCUSSION THREAD

**Discussion by vernonet** [gmallios/SoundcoreManager#140] [Q&A] [answered]
What operating systems does the Windows app support? Doesn't work for me on win 7 x64. I tried it on two PCs with Windows 7 x64 - the application does not find various libraries (for example, bluetoothapis.dll, etc.). Works fine on win 10.

**Comment by gmallios** [comment, marked as answer]
It's only tested on Windows 10/11 at the moment but i can spin up a windows 7 VM to try make it compatible.
